corehq/apps/app_manager/models.py

```diff
@@ -5707,63 +5707,89 @@ def reapply_overrides(self):
 
 
 def import_app(app_id_or_source, domain, source_properties=None, request=None, check_all_reports=True):
```

Comment:
Additional possible refactor: it looks like `check_all_reports` is never passed in by any calls to this function.

Reply:
Looks like the same is true for `source_properties`

Reply:
I think you are right about `check_all_reports` but `source_properties` is passed in here:
https://github.com/dimagi/commcare-hq/blob/9366973aba62ab189848c5e26d30e4d5a0de1af9/corehq/apps/app_manager/views/apps.py#L492
and a bit further (farther?) down in that file. That tricky `...import import_app as import_app_util`

Reply:
aha, good catch